components/molecules/CallToAction.js

```diff
@@ -8,12 +8,27 @@ import { useTranslation } from "next-i18next";
  */
 export function CallToAction(props) {
   const { t } = useTranslation("common");
+  const texts = props.description
+    .split(/(unsubscribe|désinscrire)/)
+    .filter(Boolean);
```

Comment:
I am not sure why you are using a Boolean constructor as the filter method but I think we should avoid configuring this component in a way that it's behaviour is dependent on a specific value in the string being passed in. The component should not at all be concerned with any specifics of the data. I think it's a better idea to use the html or markdown string method as it removes this logic entirely and makes it so we can pass any string in and see a predictable output.